src/core/bitop.d

```diff
@@ -72,20 +72,18 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
+{
 /// ditto
+int bsf(ulong v) pure;
+}
+else version(D_LP64) { } else
```

Comment:
> There is no `D_LP32` standard version. `size_t` is 32 bit unless `D_LP64` is set.

So `static if(size_t.sizeof == 4)` is equal to `version(D_LP64) { } else`. This is because `size_t` can be either 32 bit or 64 bit and there is no plans to support anything else (and if there were such plans something like `D_LP32` version would be added at first).